package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "iitc-ce",
+  "description": "ingress intel total conversion (IITC-CE)",
+  "keywords": [
+    "iitc",
+    "ingress"
+  ],
+  "main": "core/total-conversion-build.js",
+  "repository": "https://github.com/IITC-CE/ingress-intel-total-conversion.git",
+  "author": ".",
+  "license": "ISC",
+  "private": true,
+  "devDependencies": {
+    "eslint": "^7.5.0"
```

Comment:
Not sure about it, because in fact it is not used yet.
And current version is 7.16, so may be better just keep `^7` (no difference, but more expessive)

Reply:
could be updated, yes - when doing this PR 7.5 was latest-release. 

I don't like automatic version updates of underlying libs. (eslint might be an exception here - but I don't like exception either :) )